river/Cursor.zig

```diff
@@ -58,9 +160,11 @@ const Mode = union(enum) {
         view: *View,
         delta_x: f64 = 0,
         delta_y: f64 = 0,
-        /// Offset from the lower right corner of the view
-        offset_x: i32,
-        offset_y: i32,
+        resize_direction: ResizeDirection,
+
+        /// The distance between cursor and the nearest corner or edge.
+        offset_x: f64,
+        offset_y: f64,
```

Comment:
I don't see a reason these need to be floating point, can we just make them `i32`s?